src/views/pages/settings.ejs

```diff
@@ -0,0 +1,65 @@
+<%- include("../includes/head") %>
+    <link rel="stylesheet" href="/css/includes/footer.css">
+    <link rel="stylesheet" href="/css/components/input.css">
+    <link rel="stylesheet" href="/css/components/button.css">
+    <link rel="stylesheet" href="/css/includes/overlay.css">
+    <link rel="stylesheet" href="/css/components/link.css">
+    <link rel="stylesheet" href="/css/pages/settings.css">
+    <link rel="stylesheet" href="/css/components/navigation.css">
+    <link rel="stylesheet" href="/css/components/sidebar.css">
+    <link rel="stylesheet" href="/css/components/separator.css">
+</head>
+	<body>
+        <%- include('../components/navigation.ejs', {routePath: settingsPath})%>
+        <main class="main">
+            <div class="container">
+                <h1 class="heading">Налаштування</h1>
+                <div class="profile-settings">
+                    <div class="profile-settings__content">
+                        <div class="form-container">
+                            <h2 class="form__heading">Профіль</h2>
+                            <%- include("../components/separator.ejs") %>
+                            <form class="change-name-form">
```

Comment:
either this is just form, or name upper class names to include change-name-form-container, for example